product_warranty/models/product.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    warranty_time = fields.Integer(string='Warranty Time',
+                                   help='Number of months before the warranty '
```

Comment:
The message is misleading. It seems this is the number of months till the warranty expires, what would be a computed field. However, this field is fixed, then this is the number of months the warranty covers.

Reply:
Is in the task some point to compute the remaining months to expire the warranty? That would be a better contribution than just adding a fixed field.

Reply:
In the task, the lots, when informing the "Installation Date", uses this field to calculate when the warranty date expires.

Reply:
https://github.com/brain-tec/xplace/pull/210

Reply:
Ok, but here that functionality is not available. This is an standalone module, that for the OCA has to work without any dependence to our xplace methods. That's why we should think about that. 
Perhaps we could bring that functionality to this module

Reply:
Perhaps we should change the Remarks of the task... where those:
* The warranty time on the product should be added to OCA/product-attributes
* The warranty date on the lot and setting it automatically is part of the xplace module that implements the installation date